src/Components/Faq/Faq.js

```diff
@@ -2,7 +2,72 @@ import React from 'react';
+          Particulate matter contains microscopic solids or liquid droplets that are so small that they can be inhaled and cause serious health problems. Some particles less than 10 micrometers in diameter can get deep into your lungs and some may even get into your bloodstream. Of these, particles less than 2.5 micrometers in diameter, also known as fine particles or PM2.5, pose the greatest risk to health. Fine particles are also the main cause of reduced visibility (haze) in parts of the United States.
+        </p>
+        <h2>Difference in levels:</h2>
+        <p>
+          <span>
+          p1/PM10(particles with a diameter of 10 micrometres or less): these particles are small enough to pass through the throat and nose and enter the lungs. Once inhaled, these particles can affect the heart and lungs and cause serious health effects.
+          </span>
+          <span>
+          p2/PM2.5(particles with a diameter of 2.5 micrometres or less): these particles are so small they can get deep into the lungs and into the bloodstream. There is sufficient evidence that exposure to PM2.5 over long periods (years) can cause adverse health effects. Note that PM10 includes PM2.5.
+          </span>
+        </p>
+      </article>
+      <article>
+        <h2>Mask Facts</h2>
+        <p>
+          <b>Cotton Handkerchief</b> – A handkerchief is a small, cotton piece of cloth usually used as a clothing accessory or to blow your nose. Some choose to wear it as a face mask. Research shows that it blocks 28% of 0.007 micron particles.
+        </p>
+        <p>
+          <b>Homemade Mask</b> – A homemade mask is a mask that is sewn by an individual, usually comprised of cotton fabric. Research has shown that these types of masks filtered 69.4% of virus sized particles.
+        </p>
+        <p>
+          <b>Dust Mask</b> –A dust mask is a mask that has not been tested to any government performance standards for filtration. They are designed to block larger particles while doing activities such as mowing grass, gardening, etc. to prevent irritation to the mouth, nose, and throat. Dusk masks are not respirators and cannot protect against extremely small particulate matter. There is usually no printing on the mask. 
+        </p>
+        <p>
+          <b>Surgical Mask</b> – This type of mask is a face mask that is approved by the FDA for use as a surgical mask. They do not have printing on the mask. The mask is loose-fitting and is fluid-resistant to provide the wearer protection against large droplets, splashes, or sprays of bodily or other hazardous fluids. Surgical masks protect a patient from the wearer’s respiratory emissions. Research has shown that surgical masks filtered 80% of virus sized particles.
+        </p>
+        <p>
+          <b>N95 Mask</b> – The N95 mask is a respirator mask that is tightfitting and filters at least 95% of of particles(PM) larger than 0.3 microns.
+        </p>
+      </article>
+    </section>
+  )
```

Comment:
This is looking awesome! I really have no improvements to suggest, I think this has everything we need.